packages/common/src/core/schemas/internal/EditorOptions.ts

```diff
@@ -3,6 +3,10 @@ import { HubEntity } from "../../types";
 /** Intersection type of all EditorOptions */
 export type EditorOptions = EntityEditorOptions | CardEditorOptions;
 
+// but once I added IHubUser to HubEntity I would get errors
+// about isSharedUpdate not existing on IHubUser
+// which is odd b/c it only exists on IHubGroup
+// so, for now, I'm using a union type that excludes IHubUser
```

Comment:
unclear what this comment is related to... did you move something?

Reply:
oops, that was for some temporary code that I removed.